src/global.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
al deze JS files in de src map lijken me niet de bedoeling? Als het compiled is dan moet het in `dist/`